deploy/monai-deploy-express/init.sh

```diff
@@ -18,6 +18,13 @@ RUNDIR=$PWD/.md
 echo "Initializing directories..."
 [ -d $RUNDIR ] && echo "Removing existin $RUNDIR" && sudo rm -r $RUNDIR
 mkdir -p $RUNDIR/esdata/ && echo "Created $RUNDIR/"
-sudo chown 1000:1000 -R $RUNDIR/esdata && echo "Permission updated"
+mkdir -p $RUNDIR/minio/ && echo "Created $RUNDIR/minio/"
+mkdir -p $RUNDIR/rabbitmq/ && echo "Created $RUNDIR/rabbitmq/"
+mkdir -p $RUNDIR/orthanc/ && echo "Created $RUNDIR/orthanc/"
+mkdir -p $RUNDIR/mongodb/ && echo "Created $RUNDIR/mongodb/"
+mkdir -p $RUNDIR/mdwm/ && echo "Created $RUNDIR/mdwm/"
+mkdir -p $RUNDIR/mdtm/ && echo "Created $RUNDIR/mdtm/"
+mkdir -p $RUNDIR/mdig/ && echo "Created $RUNDIR/mdig/"
+sudo chown $(id -u):$(id -g) -R $RUNDIR/esdata && echo "Permission updated"
```

Comment:
How would this work if the container image user does NOT have the same ID as the the logged-on user running MD Express.

Also, the root cause is that the TM container creates a sub folder for the MAP, under an existing work dir for the TM. Changing the mode of the parent work dir does not affect the newly created subfolder, for mapping as output folder for the MAP container, which would still be owned by the user ID of the TM container, root, by default. That is why workaround was not provided, and we had to modify the TM/Docker Plugin.

Reply:
So this is unrelated, we have had this long ago. The change is inside Docker plug-in as linked above.

Reply:
Here's a screenshot showing the result back in Orthanc.
<img width="743" alt="image" src="https://github.com/Project-MONAI/monai-deploy/assets/4236590/0d293ae9-e383-4af9-9f54-bb708df6f5ad">

Reply:
Argh, the other PR where a sub-process is launched to chmod. OK, dots are connected.